src/views/Pipeline.vue

```diff
@@ -502,7 +502,14 @@ export default defineComponent({
         event: ev,
         componentProps: { job }
       });
-      return popover.present()
+
+      popover.onDidDismiss().then((result) => {
+        const jobEnumId = result.data.systemJobEnumId;
```

Comment:
Add a condition that whether we have data in the result or not.

Reply:
updated sir